src/Components/Play/HumanVsHuman.jsx

```diff
@@ -396,83 +400,115 @@ class HumanVsHuman extends Component {
+                                query.set('response', 'false');
+
+                                // Send the POST request to the server
+                                axios.post('https://negativei2-server.herokuapp.com/respondoffer', query)
+                                    .then(function(response) {
+                                        // Close offer window
+                                        confBox.style.display = "none";
+                                    })
+                                    .catch(function(error) {
+                                        var tmp = $('<div></div>');
+                                        tmp.html(error.response.data);
+
+                                        var message = $('p', tmp).text();
+                                        console.log(message);
+                                    });
+                            }
+                        }
 
-                    var message = $('p', tmp).text();
-                    console.log(message);
-                });
-        }
+                        // Accept the draw offer
+                        const acceptDrawBtn = document.getElementById("accept-draw");
+                        acceptDrawBtn.onclick = (event) => {
+                            query.set('response', 'true');
+
+                            // Send the POST request to the server
+                            axios.post('https://negativei2-server.herokuapp.com/respondoffer', query)
+                                .then(function(response) {
+                                    // Close offer window
+                                    confBox.style.display = "none";
+
+                                    // Go to end game page
+                                    //this.props.history.push('/endgame');
+                                })
+                                .catch(function(error) {
+                                    var tmp = $('<div></div>');
+                                    tmp.html(error.response.data);
+
+                                    var message = $('p', tmp).text();
+                                    console.log(message);
+                                });
+                        }
 
-        // Close window when "Decline" is clicked
-        const declineDrawBtn = document.getElementById("decline-offer");
-        declineDrawBtn.onclick = (event) => {
-            // Decline offer
-            query.set('response', 'false');
+                        // Close window when "Decline" is clicked
+                        const declineDrawBtn = document.getElementById("decline-offer");
+                        declineDrawBtn.onclick = (event) => {
+                            // Decline offer
+                            query.set('response', 'false');
+
+                            // Send the POST request to the server
+                            axios.post('https://negativei2-server.herokuapp.com/respondoffer', query)
```

Comment:
Should use `server.post('/respondoffer', query)` here instead of `axios`. _See #110._